packages/preview/ncku-thesis-template/0.1.0/typst.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ncku-thesis-template"
```

Comment:
Can you please rename your package? The `template` part is redundant in my opinion. But more importantly, unless this is an official template, the name of your package should include some distinctive component, so that is not the canonical name for what your package does. You can try to make a pun, or add an adjective to the current name, for example.